src/components/DiseaseTable.tsx

```diff
@@ -103,6 +105,29 @@ const DiseaseTable = ({
         ),
     };
 
+    const handleCellInHoveredRow = (
+        record: UnpackedDiseaseCellLine,
+        index: number | undefined
+    ) => {
+        const hoveredAndDataComplete =
+            index === hoveredRowIndex && record.status === "data complete";
+        return {
+            className: hoveredAndDataComplete ? hoveredRow : "",
+            onMouseEnter: () => {
+                if (index !== undefined) {
+                    setHoveredRowIndex(index);
+                }
+            },
+            onMouseLeave: () => {
+                setHoveredRowIndex(-1);
+            },
+            onClick: () => {
+                if (hoveredAndDataComplete)
+                    window.location.href = `/disease-cell-line/AICS-${record.cell_line_id}`;
```

Comment:
This doesn't seem to be causing build errors, but gatsby has a navigate function 
`import { navigate } from 'gatsby';` for this kind of thing. I'm not sure if there are advantages or not

Reply:
Ohh, thank you. I don't know why I didn't think of that. I went looking for react-router.